src/lib/classes/proxy-deployer.spec.ts

```diff
@@ -23,17 +23,17 @@ describe('UniversalProfile', () => {
     );
     await erc725AccountProxy.initialize(await signer.getAddress());
 
-    await erc725AccountProxy['setData(bytes32[],bytes[])'](
-      ['0x5ef83ad9559033e6e941db7d7c495acdce616347d28e90c7ce47cbfcfcad3bc5'],
-      ['0xbeefbeef'],
+    await erc725AccountProxy.setData(
```

Comment:
Why is this one not setDataBatch ? Does it have to do something with the proxy ?

Reply:
setData is still a function it just no longer takes an array